deployment/terraform/subnets.tf

```diff
@@ -0,0 +1,28 @@
+data "aws_subnet" "subnet1" {
```

Comment:
You can also use networking module:
```
module "network" {
  source        = "git@github.com:kids-first/aws-infra-network-module"
  organization  = var.organization
  environment   = var.environment
  subnet_prefix = "apps"
  azs           = var.azs
}
```
There are the outputs from the module:
https://github.com/kids-first/aws-infra-network-module/blob/master/outputs.tf

Reply:
```azs = ["a","b"]```

Reply:
Fixed in https://github.com/kids-first/kf-lambda-member-update/pull/15/commits/5e485fa4bb17409225b69a0a8790e9287837140f